9.4/paper.bib

```diff
@@ -496,38 +502,41 @@ @Article{trilinos
   volume =       31,
   pages =        {397--423}}
 
+See: https://trilinos.github.io/cite.html
```

Comment:
I think you want to make this a comment :-)

Reply:
Bibtex actually only distinguished between _inside_ and _outside_ of an entry. So as long as there is no '@' symbol in the comment, bibtex will ignore it. See also for example [here](https://maverick.inria.fr/~Xavier.Decoret/resources/xdkbibtex/bibtex_summary.html#comment).

If you insist, I can add the Latex comment symbol '%'  at the beginning of the line. But it really makes no difference for bibtex.

Reply:
Interesting. I had no idea :-) I suspect I'm not the only one who is confused, so if you don't mind too much, add a `%` or `#` to the beginning of the line. That might also make sure that things are rendered with a different color.